source/app/helpers/sessions_helper.rb

```diff
@@ -1,2 +1,18 @@
 module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def log_out
+    session.delete :user_id
+    @current_user = nil
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
+    ! current_user.nil?
+  end
 end
```

Comment:
You may want to use a controller concern instead, as helpers are usually used for view helper methods (though the line between them is a bit blurry).